src/core/core.scale.js

```diff
@@ -634,13 +634,26 @@ module.exports = Element.extend({
 		var cosRotation = Math.cos(labelRotationRadians);
 		var longestRotatedLabel = me.longestLabelWidth * cosRotation;
 		var result = [];
-		var i, tick, shouldSkip;
-
+		var i, tick;
+
+		// Specifiy the label spacing
+		var labelSpacing;
+		if (optionTicks.labelSpacing) {
+			labelSpacing = optionTicks.labelSpacing;
+			// If not an even number, reject
```

Comment:
Why this restriction? We should mention in the documentation if we keep the restriction

Reply:
The restriction is because the ticks are integers, updated to say 'integer' instead of 'even number'